### ISSUE TITLE
Busybox tar does not support requested options

### ISSUE THREAD

**Issue by samuelpowell** [JuliaPackaging/BinaryProvider.jl#162]
As per this [comment](https://github.com/Gowerlabs/LibFTD2XX.jl/pull/45#issuecomment-494294250), unpack on an embedded ARMv7 device using busybox fails due to options passed to tar.

```
│ [08:26:52] tar: unrecognized option '--directory=/home/root/LibFTD2XX.jl/deps/usr'
│ [08:26:52] BusyBox v1.24.1 (2019-04-01 10:26:50 UTC) multi-call binary.
```

**Comment by hhaensel** [comment]
could you run a tar and see, whether option `-C` is supported?
On my windows machine `busybox tar` tells me, it would support `-C`.

I am running busybox v1.27.0, but it supports both `-C` and`--directory=`
So upgrading busybox should do the job.

If that is not feasible than you could still replace `--directory=` by `-C` in `BinaryProvider`. In the latest master you might also have to change `--exclude-from=` by `-X` (I think tar doesn't need the space after the option, although the manual tells you, it does). But the latter is only critical, if your system doesn't understand symlinks, otherwise it will not be used.

**Comment by samuelpowell** [comment]
Thanks @hhaensel we'll try and answer that. 

@tdowrick do you still have access to the embedded board which gave us this problem? Could you try @hhaensel's suggestion of running `tar -C` or even just print out the help information so we can see what options it would support (and what version it is)?

**Comment by tdowrick** [comment]
I do, yes. Won't be able to look at it until tomorrow evening, but will let you know when done.

**Comment by samuelpowell** [comment]
Super, thanks

**Comment by hhaensel** [comment]
I have released a PR for BinaryProvider that should work




On Tue, Jun 18, 2019 at 2:11 PM +0200, "Samuel Powell" <notifications@github.com> wrote:












Super, thanks

**Comment by samuelpowell** [comment]
Thanks @hhaensel I don't have access to the target myself, but looking over the shell output in the comment linked in the original post, it appears that busybox reports supporting both `-C` and `-X`, so assuming these have always had the same meaning (and assuming this doesn't break anything) it should work for us.

**Comment by tdowrick** [comment]
Was going to do this today, but just realised I left a cable I need at UCL, so will have to wait until next week,

**Comment by samuelpowell** [comment]
Okay, let us know.

**Comment by tdowrick** [comment]
`tar -[cxtZzJjahmvO] [-X FILE] [-T FILE] [-f TARFILE] [-C DIR] [FILE]...`
You're correct that it does accept `-c` as an argument.
Sorry for the delay!

**Comment by hhaensel** [comment]
Did you try to add/build `LibFTD2XX` with my [branch `hh/normpath` of `BinaryProvider.jl`](https://github.com/hhaensel/BinaryProvider.jl/tree/hh/normpath)?

**Comment by tdowrick** [comment]
I just tried it and it succeeds.

**Comment by hhaensel** [comment]
So this could also be closed? Version 5.5 is out :-)

**Comment by samuelpowell** [comment]
Thank you very much for your help @hhaensel
